pkg/dynamicinstrumentation/testutil/exploration_e2e_test.go

```diff
@@ -0,0 +1,1206 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build exploration_test_linux_bpf
```

Comment:
Deliberately avoid being part of the build process. The exploration testing, for now, will be a tool we use locally. In future PR it will be part of the CI pipeline and kick when di related code changes